test/common/helpers/custom-resource-provider-helper.test.ts

```diff
@@ -20,9 +20,9 @@ import { buildCustomResourceProvider } from '../../../src/common/helpers/custom-
 const AOSSCRProvider = buildCustomResourceProvider({
   providerName: 'OpenSearchIndexCRProvider',
   codePath: path.join(
-    __dirname, '../../../lambda/opensearch-serverless-custom-resources'),
-  handler: 'custom_resources.on_event',
-  runtime: lambda.Runtime.PYTHON_3_12,
+    __dirname, '../../../lambda/opensearch-serverless-custom-resources/index.ts'),
+  handler: 'handler',
+  runtime: lambda.Runtime.NODEJS_18_X,
```

Comment:
Consider using latest AWS Lambda runtime?
```suggestion
  runtime: lambda.Runtime.NODEJS_20_X,
```